src/API/Trace/functions.php

```diff
@@ -35,7 +35,7 @@ function trace(SpanInterface $span, Closure $closure, iterable $args = [])
         return $c(...$a, ...($a = []));
     } catch (Throwable $e) {
         $s->setStatus(StatusCode::STATUS_ERROR, $e->getMessage());
-        $s->recordException($e, [TraceAttributes::EXCEPTION_ESCAPED => true]);
+        $s->recordException($e, [EventAttributes::EXCEPTION_ESCAPED => true]);
```

Comment:
```suggestion
        $s->recordException($e, ['exception.escaped' => true]);
```
Can we drop the `open-telemetry/sem-conv` from the API-package to avoid breaking the API on sem-conv updates?

Reply:
That's a good question. If we did drop it, what is the point of having the sem-conv package? I've noticed that other SIGs version their semantic convention package the same as the spec version (eg 1.19.0 in this case), and presumably then pin APIs etc to that version. I have a feeling that might be very painful if packages are dependant on different versions, though.

The other thing some SIGs are doing is moving all attributes into two classes: `ResourceAttributes`, and `SemanticAttributes`, the latter being "everything except resources". That should at least solve the problem of things moving about as the spec evolves.

Reply:
IMO contrib/instrumentation packages are the main consumers of the sem-conv package.
(Java added removed attributes to the template to avoid a BC-break https://github.com/open-telemetry/opentelemetry-java/commit/188210bdba24d9ba45d8fcf9edd68deacc98afd4, I like this approach because it makes pinning to a specific version in instrumentation packages obsolete when specifying the schema url as literal instead of relying on `TraceAttributes::SCHEMA_URL`.)

Regarding the API dependency: even if we resolve the BC-break I don't see a reason to keep the sem-conv dependency in the API package, we only use it for a small helper function. Iff `::recordException()` / `exception.escaped` are changed in v1 in a breaking way, we should deprecate the `trace()` function. (The SDK uses literals for the other 'exception.' attributes in `::recordException()` too / any breaking change would very likely also affect the SDK.)

Reply:
Thanks for pointing that out - adding removed items to the template and marking them as `@deprecated` is a good idea. I've loosely copied java's deprecation pattern, and decided to remove the new semver classes I've added, since java hasn't done anything yet and I'd be more comfortable if they go first and we copy.